src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -14,6 +14,22 @@ const {SALT_LENGTH, PBKDF2_ITERATIONS} = CryptoConstants;
 type Plaintext = Uint8Array;
 let RecryptApi: Recrypt.Api256;
 
+/**
+ * Augments UserPrivateKey and return both the augmented private key and the augmentation factor.
+ * @param userPrivateKey current private key to be augmented
+ * @param augmentatioFactor
+ */
+export const getRotationResults = (userPrivateKey: Uint8Array): Future<Error, {newPrivateKey: Uint8Array; augmentationFactor: Uint8Array}> => {
```

Comment:
This function name doesn't feel very clear about what it's actually doing. Maybe just `rotateUsersPrivateKey` or something instead? Maybe there's already a name used in ironoxide that we could copy.